byu-calendar-feature-links/byu-calendar-feature-links.scss

```diff
@@ -0,0 +1,142 @@
+$teal: #5d7998;
+$navy: #002e5d;
+$white: #fff;
+
+:host {
+  width: 19%;
```

Comment:
Also go through and mixin-ify this file, and have a byu-calendar-feature-links-common.scss for mixins.